### PR TITLE
Change report message

### PR DESCRIPTION
**Pull request by Thomas-AC-Dev**
Changes the report message to "Report has been sent". The old report message lies to you, and I know this is a small PR, but I think it needs to be done.

### CODE REVIEW

**Comment by ClobberXD** [comment]
Please split the commits into separate PRs.

**Comment by Thomas-AC-Dev** [comment]
Sorry. I didn't realized that it pushed that automatically from master.

File: mods/other/report/init.lua [hunk 1]
```diff
@@ -85,6 +85,6 @@ minetest.register_chatcommand("report", {
 			end
 		end
 
-		return true, "Reported. We'll get back to you."
+		return true, "Report has been sent."
```
Comment (ClobberXD) [inline mods/other/report/init.lua]: Might as well be `Reported. We'll never get back to you`...

But this one's _just as neutral_, and I'll approve once the other two commits are removed from this PR.

Comment (Thomas-AC-Dev) [reply mods/other/report/init.lua]: I am really sorry, but I don't know how to get rid of the other commits. If I can DM you on Discord for some help, that would be great. Thank you.

Comment (ClobberXD) [reply mods/other/report/init.lua]: The other commits seem to have been removed correctly. Don't worry about the number or commits. CTF always squashes the PR commits into one before merging.

Comment (Thomas-AC-Dev) [reply mods/other/report/init.lua]: Okay, thank you. I didn't know why the number of commits kept growing, I am still new to GitHub, so thank you.

**Pull request merged**